ops/terraform/env/mgmt/github-actions-iam.tf

```diff
@@ -334,13 +334,158 @@ resource "aws_iam_policy" "github_actions_static_site" {
   )
 }
 
+resource "aws_iam_policy" "github_actions_ci_ops" {
```

Comment:
@sb-benohe We're acknowledging that this policy is overly permissive. We discussed this briefly yesterday and concluded that a subsequent effort to tighten this policy is most appropriate. Pending your feedback and identification of that subsequent effort in JIRA, I think we might be able to accept this.

Reply:
I pulled all Cloudwatch log errors regarding permission failures during 9.19 testing runs, will provide for new ticket to assist in analysis.

Reply:
Thanks this is definitely the direction we would want to move. The other thing we will have to keep top of mind is making sure ALL of our pushes to MASTER are in a "deployable" state as anything in a "non-deployable" state could cause _chaos_ . Also making sure no outside contributors get access to this without through review as this is essentially a door to our AWS env.